tests/state/test_state_api.py

```diff
@@ -31,7 +31,7 @@ def test_invalid_state_message(self, stub_broker, api_client):
 
     @pytest.mark.parametrize("name_state", [StateNamesEnum.Skipped, StateNamesEnum.Success])
     def test_get_state_by_name(self, name_state, stub_broker, state_middleware, api_client):
-        state = State("3141516", name_state, [], {})
+        state = State("3141516", name_state)
```

Comment:
I would be nice if you tested the method set_state of state backend. as it's starting to be complex.